README.md

```diff
@@ -136,6 +136,25 @@ class Comment < ActiveRecord::Base
 end
 ```
 
+## Addition Project Configurations
+Additional global configurations are available to match a developer's project conventions.
+Similar to setting the `character set` and `collation` at the global level you can also configure the
+default `text limit`, `string limit`, `null`, `generate foreign keys`, and `genereate indexing`.
```

Comment:
I think it will be simpler to understand if these options are just grouped with the character set ones in a "Configuration" section. No need for one to reference the other.

Reply:
Please make a list (could be a ### section) that lists them out and has room for a sentence or two about each. I'd just repeat the rule about what `nil` means for the 3 cases where it's supported. Ideally, document the exception that will be raised if they break that rule.